### ISSUE TITLE
attached wallet fallbacks on failure in order of priority

### ISSUE THREAD

**Issue by huumn** [stackernews/stacker.news#1494]
For zaps, this is more complicated than it sounds because either party in a p2p zap can cause it to fail and we only want to "fallback" for the failing side.

Also, if all fallbacks fail for a zap, should the last resort be cowboy credits?

**Comment by ekzyis** [comment]
> Also, if all fallbacks fail for a zap, should the last resort be cowboy credits?

This is how it currently works in #1688. I think we mentioned this might be a setting but it's not for now.

**Comment by huumn** [comment]
I don't think it will end up needing to be a setting, but we can see how people react over time.
